### PR TITLE
[WEB-2477] add mrn/ehr settings api and form validation

### PR DESCRIPTION
**Pull request by krystophv**
for [WEB-2477], adds clinic MRN/EHR settings endpoints, fetches on clinic fetch, and enforces MRN requirement as necessary

relies on: https://github.com/tidepool-org/platform-client/pull/151
relates to: https://github.com/tidepool-org/uploader/pull/1578

[WEB-2477]: https://tidepool.atlassian.net/browse/WEB-2477?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by clintonium-119 — CHANGES_REQUESTED** [review]
Just one small translation key suggestion that I _think_ we need before approval.  Otherwise, looks great :+1:

File: app/components/clinic/PatientForm.js
Comment (krystophv) [reply app/components/clinic/PatientForm.js]: Ah yeah, even if it _did_ work, it'd almost certainly be inviting some sort of edge case behavior. Good call 👍🏼

**Review by clintonium-119 — APPROVED** [review]
LGTM :shipit:

**Comment by krystophv** [comment]
Supplanted by https://github.com/tidepool-org/blip/pull/1289

**Pull request closed without merging**
